packages/terra-form-select/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Added
+* A SelectField component.
```

Comment:
Nit: The log looks a little weird with just the sentence. I recommend changing A to Added

Reply:
It does feel a little strange, but it's also strange to double the word Added under a section called Added. Any ideas to make the sentence read cleaner without doubling the same section word?

Reply:
Updated to read as: 
`* SelectField - A convenience component that wraps a select within a field.`